Makefile.config

```diff
@@ -23,6 +23,23 @@ else
         $(info Note: Building without GnuTLS support)
 endif
 
+ifeq (x$(call pkg-config-check,libnftables),xy)
```

Comment:
Sorry in advance as I have small experience in makefiles, but why not just ?
```ifeq ($(call pkg-config-check,libnftables),y)```

Reply:
As far as I know, very old shells did not handle empty strings correctly in comparisons. Nowadays it's just a tradition. :)

Reply:
On line 13 we have
```ifeq ($(call pkg-config-check,libselinux),y)```
with no problem somehow, why?

Reply:
I've fixed that to avoid meaningless holywar. :)

Reply:
I agree, that if we use `ifeq` without `x` in upper code, we should do that uniformly in all codebase.